nuxeo-runtime/nuxeo-runtime-metrics/src/main/java/org/nuxeo/runtime/metrics/MetricsDescriptor.java

```diff
@@ -109,6 +139,19 @@ public String prefix() {
             return prefix.replace("${hostname}", hostname);
         }
 
+        protected boolean filter(String name) {
+            if (filters == null || filters.isEmpty()) {
+                String filters = Framework.getProperty(FILTER_PROPERTY, DEFAULT_METRICS);
```

Comment:
- The local variable `String filters` is the same name as your `List<String> filters` (which I find confusing).  

- How often is filter() method called?  Do you want to set your `List<String> filters` with the `filters.split()` values so you don't need to do it every time.